### PR TITLE
Upgrading logkitty package

### PR DESCRIPTION
**Pull request by dalmendray**
Fixing the issue: https://github.com/react-native-community/cli/issues/1175

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@react-native-community/eslint-config": "^0.0.3",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^25.2.6",
+    "babel-plugin-macros": "^2.8.0",
```
Comment (tcirstea) [inline package.json:37]: Moving this line seems unrelated?

File: package.json [hunk 2]
```diff
@@ -69,5 +69,8 @@
     "packages": [
       "packages/*"
     ]
+  },
+  "dependencies": {
+    "logkitty": "^0.7.1"
```
Comment (tcirstea) [inline package.json:74]: Im not sure this actually fixes it - if you look at yarn.lock,  logkitty@^0.6.0 is still installed and use separately.

**Review by tcirstea — COMMENTED** [review]
What's the deaul with package-lock.json? I don't see that mentioned anywhere

File: package.json (hunk 1 above)
Comment (dalmendray) [reply package.json:37]: Correct, this was autofixed when running yarn

Comment (dalmendray) [reply package.json:37]: As for the fix, I updated the CLI following the steps here: https://github.com/react-native-community/cli#updating-the-cli

File: package.json (hunk 2 above)
Comment (dalmendray) [reply package.json]: As for the fix, I updated the CLI following the steps here: https://github.com/react-native-community/cli#updating-the-cli

**Comment by dalmendray** [comment]
> What's the deaul with package-lock.json? I don't see that mentioned anywhere

In the previous commit I updated logkitty using npm and that generated the package-lock.json. With the new solution of upgrading the cli there's no need for the package-json.

**Review by apbuteau — APPROVED** [review]
